### DISCUSSION TITLE
Add tooltips to every button without text (icons, images)

### DISCUSSION THREAD

**Discussion by FluffyDiscord** [solidtime-io/solidtime#245] [Feature requests]
As a first time user (eg. easier onboarding), I would appreciate if buttons with icons or images would have short tooltip with information on what they actually do. Right now hovering over them does not show anything.

**Comment by Onatcer** [comment]
thanks for the input. Yes I agree Tooltips would help, I think especially with the the billable button it should be more clear. If there are more suggestions for Tooltips feel free to write them below, I'll triage them after and close this ticket once tooltips are implemented.
